backend/src/apiserver/resource/resource_manager.go

```diff
@@ -817,7 +821,7 @@ func (r *ResourceManager) CreateDefaultExperiment() (string, error) {
 	}
 
 	// Create default experiment
-	defaultExperiment := &model.Experiment{
+	defaultExperiment := &api.Experiment{
 		Name:        "Default",
 		Description: "All runs created without specifying an experiment will be grouped here.",
 	}
```

Comment:
There is no namespace reference here, so it will always fail in multi user mode.

Shall we add a check earlier? We just skip trying to create a default namespace when in multi user mode?

Reply:
I don't fully remember your design about default experiment. If default experiment doesn't contain a namespace, it will be useless in multi user mode.

Maybe we should just throw an error message here for multi user mode like `Please specify an experiment. No default experiment has been set up.`?

Reply:
The plan is:
1. for short term, we provide a SQL script to help users backfill existing experiments including the default one.
1. long term, we will automate the backfill in api-server using the namespace specified by a config  map/environment variable. 
So allowing the default experiment without namespace is okay for now.